core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/script/ScriptController.java

```diff
@@ -70,19 +96,21 @@ public ScriptDto post(@Valid @RequestBody ScriptPostDto scriptPostDto) throws Me
         for (ScriptPostDto scriptPostDto : scriptDtos) {
             halMultipleEmbeddedResource.embedResource(scriptPostDto);
         }
-
+        halMultipleEmbeddedResource.add(
+                linkTo(methodOn(ScriptController.class)
+                        .getAll(PageRequest.of(0, 20), new ArrayList<>(), ""))
+                        .withRel("scripts"));
```

Comment:
what does this link represent/link to?

Reply:
The entry point on when you reach /scripts

It is exactly that: /scripts?page=1&size=20

Reply:
yes, but why add this link to the return of an bulk update?

Reply:
Well, I felt like &it's about Scripts that are inserted/updated and thus are available in the API and that all returned response should (or at least that's what I understood) contain hyperlink to browse through the items of the API.